.gitattributes

```diff
@@ -2,3 +2,4 @@
 *.md export-ignore
 tests/* export-ignore
 docs/* export-ignore
+tests/Fixtures/App/var* export-ignore
```

Comment:
This one gets overwritten by `dev-kit` on the next run:
https://github.com/sonata-project/dev-kit/blob/master/project/.gitattributes.twig

Reply:
please add this to the dev-kit PR, too